terraform/README.md

```diff
@@ -0,0 +1,61 @@
+# Janus Terraform module
+
+We use these Terraform modules to manage deployments of Janus to cloud and local dev environments. We provide a distinct main module for each so that we can use different [Terraform backends](https://www.terraform.io/language/settings/backends) and configure other modules separately.
+
+A Janus environment `foo` is defined by a file `variables/foo.tfvars` which contains configuration for that environment. Among other behaviors, the variables files specify whether the environment is deployed to Google Kubernetes Engine (GKE) or to a local `kind` cluster. To see what Terraform wants to do to environment foo, run:
+
+    ENV=foo make plan
+
+To apply those changes, run:
+
+    ENV=foo make apply
+
+You may need additional environment variables or other context to interact with an environment via Terraform. See [Authentication for Terraform](https://github.com/abetterinternet/isrg-cloud-bootstrap#authentication-for-terraform) for details, as well as the sections below relevant to the environment you are working with.
+
+## Google Kubernetes Engine
+
+Staging and production environments of Janus are deployed into [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine). This uses `main_gcp/main.tf`. Variables files for GKE environments should include the variable `state_bucket`, whose value should be the name of a GCS bucket to use with the [`gcs` Terraform backend](https://www.terraform.io/language/settings/backends/gcs).
+
+[TODO: adapt prio-server/cluster-bootstrap to bringup a GKE cluster for Janus and document how to do that here]
+
+## Local development with Kind
+
+To enable local development and testing in CI, we use [`kind`](https://kind.sigs.k8s.io/) (Kubernetes IN Docker). Kind uses Docker containers to run a single-node Kubernetes control plane and optionally additional worker nodes. This uses `main_local/main.tf`. Variables files for local environments may include the `tfstate_path` variable, whose value is used as the `path` value in the [`local` Terraform backend](https://www.terraform.io/language/settings/backends/local).
+
+You will need the `kind` to manage clusters. See [the `kind` documentation](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) for instructions on installing. The quick start guide also has instructions on [creating a cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster). By default `kind` creates a cluster where a single node hosts both the control plane and workers. If you want to create a more complex cluster topology, you can use a configuration file like this:
```

Comment:
```suggestion
You will need to install `kind` to manage clusters. See [the `kind` documentation](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) for instructions on installing. The quick start guide also has instructions on [creating a cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster). By default `kind` creates a cluster where a single node hosts both the control plane and workers. If you want to create a more complex cluster topology, you can use a configuration file like this:
```

Reply:
> Count this as a vote for a `janus_ops` repository.

SGTM! I will close this PR and will create a new, private repo for Terraform and config.